pages_/scan.tsx

```diff
@@ -53,6 +53,14 @@ const Scan = () => {
     }
   };
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      Router.push('/');
+    }, 10_000);
+
+    return () => clearInterval(interval);
+  }, []);
+
```

Comment:
I believe this is confusing, if we are not going to ship the scanner during the pilot it should be hidden from the home screen.

Reply:
Almost all apps I have tried, they have a timeout for the cam scan, I think is a common use case.

Reply:
Why a timeout? Like I only have 10 seconds to scan the QR code? If I have a low-budget cellphone that takes a lot of time to focus what happens?

Reply:
timeout removed and cancel button added instead.